src/main/java/org/example/Proficiencies.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class Proficiencies {
+    public final String userURL = "https://www.dnd5eapi.co/api/races/:index/proficiencies";
+    public final HttpClient client = HttpClient.newHttpClient();
+    public HttpResponse<String>getProficiencies(String index) {
```

Comment:
nitpick: please introduce spaces in method declaration (ctyrl + alt + l)

Reply:
Done